scripts/unfoldv2.py

```diff
@@ -92,6 +94,10 @@ def unfold(**parsed_args):
         data_reweighter = rw,
         weight_type = parsed_args["weight_type"]
         )
+    
+    scheduler_args = json.loads(parsed_args["scheduler_args"]) if parsed_args["scheduler_args"] is not None else None
+
+    init_lr_scheduler(parsed_args["learning_rate"], parsed_args["scheduler_names"], scheduler_args)
```

Comment:
> One general comment: it would be great if we could better encapsulate the learning rate & scheduler stuff. For example, I'm not sure about the `reduce_on_plateau` argument passed in multiple levels at many places. Would it help if we wrap all relevant stuff into a `LRScheduler` (or other better names), and make it a member of `OmniFoldTTbar`? Or make it global?

Good point, it is pretty tedious. I'll combine the options into a single config file and handle all initialization inside lrscheduler class directly. (the same approach as `preprocessor`)